volumina/config.py

```diff
@@ -26,6 +26,7 @@
 [volumina]
 pixelpipeline_verbose: false
 show_3d_widget: true
+disable_gl_viewports: false
```

Comment:
Not sure about variable name here: double negation might be confusing (disable + false). On the other hand, this variable won't be used often, so maybe it's OK.

Reply:
hehe, I first had it as `enable_gl_view`... would be more natural in code I agree, but setting env vars to `0` feels weird. Usually you check whether it is set or not...